client/task-list/tasks/payments/RemotePayments/components/PaymentMethod.js

```diff
@@ -71,6 +81,12 @@ export const PaymentMethod = ( {
 						recordEvent( 'tasklist_payment_install_method', {
 							plugins,
 						} );
+
+						if ( postInstallScript ) {
+							const script = document.createElement( 'script' );
+							script.src = `${ window.location.origin }/${ postInstallScript }`;
```

Comment:
I _think_ this is okay given that the plugin defines this.  But we should probably not use `origin` since the WP install path is not always at the root.  Instead we can allow `postInstallScript` to simply define the full URL.

@psealock can I get a sanity check on the safety here?

Reply:
Right, I wondered about that very thing. I'll remove the origin and expect the full URL, although this will be obsolete soon anyways.